src/options/smt_options.toml

```diff
@@ -395,6 +395,9 @@ name   = "SMT Layer"
 [[option.mode.BV]]
   name = "bv"
   help = "Translate bvand back to bit-vectors"
+[[option.mode.BITWISE]]
```

Comment:
On the one hand, we need this option to merge the new code to `int_blaster.cpp`. On the other hand, what is actually running right now is `bv_to_int.cpp`, which does not acknowledge this option. I suggest, for now, adding a check to `bv_to_int.cpp` that throws a meaningful error to the user (`bitwise option is not supported currently`, or something), in case someone (or some testing tool) tries this option.

Reply:
Good point! Added with 31c246889261b9fc6c862e1131acc248b523c641. This made me realize that there are now two `BITWISE` options: one for `solve-bv-as-int` and one for `iand-mode`. Did I do this correctly? In this PR do I actually need the `BITWISE` option for `solve-bv-as-int` (which I just added), or should I just be checking the `wand-mode` (which already has a [bitwise option](https://github.com/cvc5/cvc5/blob/1339d73750e15422a5c2085c4819c15b77bededd/src/options/smt_options.toml#L421)). Sorry for the dumb questions! Still paging this back in.

Reply:
31c2468 looks great, thanks.

You are correct in everything you pointed out:
1. `solve-bv-as-int=bitwise` === eager-bitwise
2. `solve-bv-as-int=iand --iand-mode=bitwise` === lazy-bitwise
3. The code is relatively modular, and so the lazy bitwise option (bitwise in iand mode) is already tested: https://github.com/cvc5/cvc5/blob/70997d0e3ebf2027279373d9594c66119f3fa656/test/regress/regress3/nl/iand-native-1.smt2#L3 (notice that the test was added by you :) https://github.com/cvc5/cvc5/pull/5328). True, we still don't have a unit test for the iand solver, but I think that this is out of scope for the current PR.

So what you implemented and tested in this PR is eager-bitwise.